src/lib/ext-monitor.js

```diff
@@ -97,7 +106,31 @@ export default class ExtensionMonitor {
     }
   };
 
+  onInstalledExtension = async (extInfo) => {
+    const { enabled, id, type } = extInfo;
+
+    const self = await browser.management.getSelf();
```

Comment:
Use `browser.runtime.id` instead of `browser.management.getSelf().id`

(you can also use this simpler method in the `getAllExtensions` method)